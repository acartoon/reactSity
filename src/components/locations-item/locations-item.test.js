import React from "react";
import renderer from "react-test-renderer";
import LocationsItem from "./locations-item.jsx";

it(`LocationsItem component render correct`, () => {
  const tree = renderer
      .create(
          <LocationsItem
            location = {`Amsterdam`}
            active = {false}
            locationClickHandler = {jest.fn()}
          />
      )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
