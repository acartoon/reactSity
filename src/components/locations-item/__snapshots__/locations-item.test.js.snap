// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationsItem component render correct 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item "
    href="#"
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
